sceptre/plan/actions.py

```diff
@@ -465,8 +465,24 @@ def create_change_set(self, change_set_name):
                 {"Key": str(k), "Value": str(v)} for k, v in self.stack.tags.items()
             ],
         }
+
         create_change_set_kwargs.update(self.stack.template.get_boto_call_parameter())
         create_change_set_kwargs.update(self._get_role_arn())
+
+        try:
+            self._create_change_set(change_set_name, create_change_set_kwargs)
+        except Exception as err:
+            self.logger.info(
+                "%s - Failed creating Change Set '%s'\n%s",
+                self.stack.name,
+                change_set_name,
+                err,
+            )
+
+    def _create_change_set(self, change_set_name, create_change_set_kwargs):
+        """
+        Wrap create_change_set.
```

Comment:
i don't understand this doc string.   isn't this more of a private helper than something that `wraps`? can you please explain?

Reply:
Oops yes, those are just placeholders. Will update!

Reply:
Based on doc strings in the rest of this file (e.g. _fetch_original_template_stage, _get_template_summary) these private methods whose behaviour is self-evident do not have doc strings at all, so I have removed these.